README.md

```diff
@@ -29,7 +29,7 @@ Installing the Provider
 
     ```sh
     $ mkdir -p $HOME/.terraform.d/plugins
-    $ mv terraform-provider-gsuite $HOME/.terraform.d/plugins/terraform-provider-gsuite
```

Comment:
We should alter the build output, otherwise it won't have a `gsuite_vX.Y.Z` here to copy